Added RPi Touchscreen device event reader

AudioMorphology opened the pull request:
Added a touch interface device reading thread, that reads the event stream from a Touch Interface on a Raspberry Pi that has been filtered, scaled & de-jittered by tslib and presented as an input device by ts_uinput.

raysan5 commented:
Thank you very much @AudioMorphology! Just added raylib forum post to [raylib Wiki](https://github.com/raysan5/raylib/wiki/Raspberry-Pi-Touch-Inputs) (still requires some formatting review)

AudioMorphology commented:
You are welcome!

I have edited the raylib Wiki to make the language more appropriate, and 
also added a page explaining how to install and configure a Touchscreen 
on a Raspberry Pi using tslib and ts_uinput.

I hope that's OK?

If you can send me your address, I will post you a small touchscreen so 
you can test this stuff on a Raspberry Pi.

Richard

On 05/01/2017 22:07, Ray wrote:
>
> Thank you very much @AudioMorphology 
> <https://github.com/AudioMorphology>! Just added raylib forum post to 
> raylib Wiki 
> <https://github.com/raysan5/raylib/wiki/Raspberry-Pi-Touch-Inputs> 
> (still requires some formatting review)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub 
> <https://github.com/raysan5/raylib/pull/214#issuecomment-270771779>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/AFz73XGpRSCnRo-PNewFKrgr34czFSDTks5rPWm-gaJpZM4LcKXW>.
>

raysan5 commented:
Thank you very much for everything @AudioMorphology! :)

Just send me an email to raysan5@gmail.com and I'll send you the address.

Regards,

Ray
